tools/resgen/src/main.cpp

```diff
@@ -245,12 +245,12 @@ int main(int argc, char* argv[]) {
     ostringstream headerStream;
     headerStream << "#ifndef " << packagePrefix << "H_" << endl
             << "#define " << packagePrefix << "H_" << endl << endl
-            << "#include <stdint.h>" << endl << endl
+            << "#include <stdint.h>\n"
+            << "#include <stddef.h>\n" << endl
```

Comment:
do we still need stddefs if you're no longer using `size_t`?

Reply:
🦅 👀